.github/linters/mlc_config.json

```diff
@@ -41,6 +41,9 @@
     },
     {
       "pattern": "^https://helpdesk.egi.eu"
+    },
+    {
+      "pattern": "^https://aai.*.egi.eu/api/v2/VoMembers"
```

Comment:
Why this "aai.*.egi.eu" and not this one "aai-*.egi.eu"?

Reply:
As it's a pattern the `.` may cover any characters, including a dash.
What is sure is that the automated link check is working properly, no errors are reported.

Reply:
@gwarf should i take any action or this approach is acceptable?

Reply:
For me it's fine. But the final review and PR validation is to be done by @vardizzo-lab.